pkg/utils/secrets/manager/manager.go

```diff
@@ -51,6 +51,10 @@ const (
 	// data is valid. In case the data contains a certificate it is the time part of the certificate's 'not after'
 	// field.
 	LabelKeyValidUntilTime = "valid-until-time"
+	// LabelKeyRenewAtTime is a constant for a key of a label on a certificate secret describing the time when the
+	// certificate should be renewed. This is an optional label to allow early rotation before the default
+	// renewal after 80% or 10d before end of validity.
+	LabelKeyRenewAtTime = "renew-at-time"
```

Comment:
```suggestion
	LabelKeyRenewAfterValidityPercentage = "renew-after-validity-percentage"
```

Wouldn't it be a nice user experience if I just had to specify the validity percentage here (`80` is there default) instead of the exact duration? 👀